php/config.yml

```diff
@@ -29,16 +28,19 @@ versions:
     build_args:
       <<: *build_args
       MUSL_VERSION: 1.2.3-r3
+      ICONV_VERSION: 1.15-r3
```

Comment:
please sort alphabetically :) (and so for 8.1 and 8.2)

Reply:
c est fait